test/issue_fail.test.js

```diff
@@ -0,0 +1,44 @@
+const nock = require('nock')
+const myProbotApp = require('..')
+const { Probot } = require('probot')
+
+const payloadFail = require('./fixtures/payloadFail.json')
+
+// 실제 주소와 연결을 하지 않고 테스트 진행
+nock.disableNetConnect()
+
+describe('Start Pull Request Open', () => {
+  let probot
```

Comment:
```suggestion
  const probot = new Probot({})
```

위와 같이 probot 변수에 초기화를 진행해두고 해도 코드 상 문제가 없어보이는데 어떻게 생각하시나요?

Reply:
현재는 1개의 테스트파일에 1개의 테스트가 존재하여 가능하나, 여러개의 테스트가 하나의 테스트파일에 들어갈 경우 정상 작동하는지 확인을 하고 변경하는게 좋아보입니다. 😄